packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/features/zoomlevel/ZoomLevelFeatureTest.java

```diff
@@ -40,7 +40,7 @@ public void before() {
     mockSensorArray = mock(Rect.class);
 
     mockedStaticCameraZoom
-        .when(() -> ZoomUtils.computeZoom(anyFloat(), any(), anyFloat(), anyFloat()))
+        .when(() -> ZoomUtils.computeZoomRect(anyFloat(), any(), anyFloat(), anyFloat()))
         .thenReturn(mockZoomArea);
   }
 
```

Comment:
Do you think we can add a test here that utilizes Robolectric to make sure the proper `CaptureRequest` is sent when `updateBuilder` is called?

Reply:
So Robolectric has some major issues mocking this call 

`mock(CaptureRequest.Builder.class);`

Seems it can only be mocked using our standard test runner - checking the rest of the project nothing mocks this while using mockito. 

I did however create a small method to help mock which version of the SDK_INT is running in our code and was able to write a few more tests using this to set our build SDK to R or Q depending on the test.

Reply:
Oh gotcha. Okay thanks!